webpack/assets/javascripts/react_app/components/common/Fill/index.js

```diff
@@ -0,0 +1,24 @@
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+import * as actions from './FillActions';
+import reducer from './FillReducer';
+
+import Fill from './Fill';
+
+// map state to props
+const mapStateToProps = state => ({
+  components: state.extendable,
```

Comment:
Don't we want to use a selector so we can reuse the statment `state.extendable`?